prihlas-me.php

```diff
@@ -13,24 +13,24 @@
 if (isset($_POST['jmeno']) && isset($_POST['heslo'])) {
     $_SESSION['pokusu_prihlaseni']++;
 
-    echo "Toto je ".$_SESSION['pokusu_prihlaseni'].". pokus o přihlášení.";
+    $_SESSION["status-prihlaseni"] = 403;
 
     $uzivatelskeJmeno = strtolower($_POST['jmeno']); // u jména nechceme řešit velikost písmen
     $uzivatelskeHeslo = $_POST['heslo'];
 
     // kontrola prihlaseni
     foreach ($admini as $admin) {
         if ($admin['jmeno'] == $uzivatelskeJmeno && $admin['heslo'] == $uzivatelskeHeslo) {
+            $_SESSION["status-prihlaseni"] = 200;
             $_SESSION['uzivatel'] = [
                 "jmeno" => $uzivatelskeJmeno,
             ];
             $uzivatelPrihlaseny = true;
         }
     }
 
-    if (!isset($_SESSION['uzivatel'])) {
-        echo "Špatně zadané uživatelské jméno nebo heslo.";
-    }
+    // redirect
+    header("Location: ".$_SERVER['HTTP_REFERER'], true, 302);
```

Comment:
Je třeba ty další parametry headeru? Je třeba `$_SERVER['HTTP_REFERER']`? Napsal bych tam to jméno kam redirectovat natvrdo, tohle vypadá jako magie.